UKHomeOfficeForms/hof
# Feature/add hods theme

# Description
Current HOF framework enables the use of `govUK` theme which implements the components and styling designed in the [Government Design System](https://design-system.service.gov.uk/). This update enables the use of the [Home Office Design System](https://design.homeoffice.gov.uk/) by implementing the relevant styles and page template.

## Changes
1. All styling changes are described on the [HODS website](https://design.homeoffice.gov.uk/)
2. The [`TaskList` component](https://design-system.service.gov.uk/patterns/task-list-pages/) has been added
3. Document title has been changed to  `{{pageTitle}} - {{appTitle}}`
4. Build task now includes copying over font assets if they exist

# Testing
1. Import this fork as `hof` package in your app
2. In `hof.settings.json`, set `"theme": "hods",`
3. Run `npm i`, then `npm run start` to start your app
4. Observe that the changes listed above are implemented
